api/turing/service/ensembling_job_service.go

```diff
@@ -142,6 +173,21 @@ func (s *ensemblingJobService) List(options EnsemblingJobListOptions) (*Paginate
 		return nil, err
 	}
 
+	if options.ProjectID != nil {
+		for _, r := range results {
+			project, err := s.mlpService.GetProject(*options.ProjectID)
```

Comment:
Same here. Why do you need to use `options.ProjectID` instead `r.ProjectID`?

Reply:
Ok changed it.